tests/foreman/ui_airgun/test_organization.py

```diff
@@ -293,3 +294,60 @@ def test_positive_update_location(session):
         assert len(org_values['locations']['resources']['assigned']) == 0
         assert location.name in org_values[
             'locations']['resources']['unassigned']
+
+
+@skip_if_not_set('fake_manifest')
+@tier2
+@upgrade
+def test_positive_delete_with_manifest_lces(session):
+    """Create Organization with valid values and upload manifest.
+    Then try to delete that organization.
+
+    :id: 851c8557-a406-4a70-9c8b-94bcf0482f8d
+
+    :expectedresults: Organization is deleted successfully.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    org = entities.Organization().create()
+    upload_manifest_locked(org.id)
+    with session:
+        session.organization.select(org.name)
+        session.lifecycleenvironment.create({'name': 'DEV'})
+        session.lifecycleenvironment.create(
+            {'name': 'QE'},
+            prior_entity_name='DEV',
+        )
+        # Org cannot be deleted when selected,
+        # So switching to Default Org and then deleting.
+        session.organization.select(DEFAULT_ORG)
+        session.organization.delete(org.name)
+        assert not session.organization.search(org.name)
+
+
+@skip_if_bug_open('bugzilla', 1651981)
+@skip_if_not_set('fake_manifest')
+@tier2
+@upgrade
+def test_positive_download_debug_cert_after_refresh(session):
+    """Create organization with valid manifest. Download debug
+    certificate for that organization and refresh added manifest for few
+    times in a row
+
+    :id: 1fcd7cd1-8ba1-434f-b9fb-c4e920046eb4
+
+    :expectedresults: Scenario passed successfully
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    org = entities.Organization().create()
+    upload_manifest_locked(org.id)
```

Comment:
Do you know that you need to take original manifest?

Reply:
Didn't notice that one manifest_upload was uploading smth different, unlike million other cases, thanks.